build/build-mrt.yml

```diff
@@ -13,6 +13,8 @@ steps:
     arguments: /envonly $(buildPlatform)fre
     modifyEnvironment: true
 
+- template: AzurePipelinesTemplates\WindowsAppSDK-InstallWindowsSDK-Steps.yml
+
```

Comment:
This change is coming from main (through a merge) and thus, it shouldn't show up here.

Reply:
Solved